sdk/storage/storage-datalake/package.json

```diff
@@ -20,6 +20,7 @@
   "module": "./esm/dataLakeStorageClient.js",
   "types": "./esm/dataLakeStorageClient.d.ts",
   "devDependencies": {
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
```

Comment:
`@azure/storage-datalake` is a Track 1 auto generated package and not a Track 2 package. The `rush lint` command should not have run the linter for this in the first place.

Reply:
oops, fixed